### ISSUE TITLE
Add support for webp

### ISSUE THREAD

**Issue by michaelherger** [andygrundman/Image-Scale#22]
webp is what all the cool kids are using nowadays 😁

**Comment by andygrundman** [comment]
Good call, and the API looks simple enough and supports some nice memory-friendly features like pre-scaling. I’ll take a look at it.

**Comment by michaelherger** [comment]
That would be great!!
